### PR TITLE
Fix flake8 noqa: `_minhash_encoder`

### PR DESCRIPTION
**Pull request by juanitorduz**
Partially closes https://github.com/skrub-data/skrub/issues/755

See https://github.com/skrub-data/skrub/pull/765

### CODE REVIEW

File: skrub/_minhash_encoder.py [hunk 1]
```diff
@@ -21,12 +21,9 @@
 NoneType = type(None)
 
 
-# Ignore lines too long, first docstring lines can't be cut
-# flake8: noqa: E501
-
-
 class MinHashEncoder(TransformerMixin, BaseEstimator):
-    """Encode string categorical features by applying the MinHash method to n-gram decompositions of strings.
+    """Encode string categorical features by applying the MinHash method to n-gram
```
Comment (Vincent-Maladiere) [inline skrub/_minhash_encoder.py:25]:
```suggestion
    """Encode string categorical features by applying the MinHash method to n-gram \
```

**Review by Vincent-Maladiere — COMMENTED** [review]
Hey @juanitorduz, you have an error in this PR —see the suggestion. You can run the docstring test locally with `pytest -s skrub/tests/test_docstrings.py`

**Comment by juanitorduz** [comment]
ok! thanks for the input! I will make the changes if needed 👍

File: skrub/_minhash_encoder.py [hunk 2]
```diff
@@ -341,7 +339,7 @@ def transform(self, X: ArrayLike) -> NDArray:
         # Handle missing values
         missing_mask = (
             ~(X == X)  # Find np.nan
-            | (X == None)  # Find None. Note: `X is None` doesn't work.
+            | (X is None)  # Find None. Note: `X is None` doesn't work.
```
Comment (Vincent-Maladiere) [inline skrub/_minhash_encoder.py:342]: This raises an error and must be reverted. See also the inline comment on the right.

Comment (juanitorduz) [reply skrub/_minhash_encoder.py:342]: OMG! I am sorry! I will revert it! (it just looks odd hehe)

Comment (juanitorduz) [reply skrub/_minhash_encoder.py]: reverted in https://github.com/skrub-data/skrub/pull/773/commits/795ba3546f26420cbcb943783fbe82452af242ca

Comment (juanitorduz) [reply skrub/_minhash_encoder.py]: Added in https://github.com/skrub-data/skrub/pull/773/commits/5a52b2011b50af638bc20a31d1cdbcf92d64b456

**Comment by juanitorduz** [comment]
> Hey @juanitorduz, you have an error in this PR —see the suggestion. You can run the docstring test locally with `pytest -s skrub/tests/test_docstrings.py`

Now these test are passing locally ✅

File: skrub/_minhash_encoder.py (hunk 2 above)
Comment (Vincent-Maladiere) [reply skrub/_minhash_encoder.py]: > OMG! I am sorry! I will revert it! (it just looks odd hehe)

Yes, there is room for improvement. Feel free to suggest enhancement in subsequent PRs or open an issue if you see fit :)

Comment (juanitorduz) [reply skrub/_minhash_encoder.py]: ok! I will do so 👍

**Pull request merged**